oak/common/app_config_serializer.cc

```diff
@@ -88,6 +89,12 @@ int main(int argc, char* argv[]) {
     }
   }
 
+  // Check application configuration validity.
+  if (!oak::ValidApplicationConfig(*config.get())) {
+    LOG(INFO) << "Application config is not valid";
```

Comment:
Why is this INFO but the other one is QFATAL?

Reply:
Done